src/pe_reports/pages.py

```diff
@@ -1,213 +1,313 @@
-        col_names = ["Sources"]
-        chart_type = XL_CHART_TYPE.COLUMN_STACKED_100
-        size = "small"
-        x, y, cx, cy = Inches(8.25), Inches(2.9), Inches(4.5), Inches(2.0)
-        stacked = False
-        Pages.insert_chart(
-            slide, df_loc, col_names, chart_type, size, x, y, cx, cy, stacked
-        )
+def buildAppendixList(df):
+    """Build report appendix."""
+    html = "<div> \n"
 
-        # Bar Graph - inferred vulnerabilities found via external observation
-        # TODO: Remove hard-coded file locations
-        # Issue 45: https://github.com/cisagov/pe-reports/issues/45
-        df_loc = "src/pe_reports/data/csv/dhs_iv_df.csv"
-        col_names = ["Sources"]
-        chart_type = XL_CHART_TYPE.COLUMN_STACKED_100
-        size = "small"
-        x, y, cx, cy = Inches(8.25), Inches(4.9), Inches(4.5), Inches(2.0)
-        stacked = False
-        Pages.insert_chart(
-            slide, df_loc, col_names, chart_type, size, x, y, cx, cy, stacked
+    for row in df.itertuples(index=False):
+        html += """<p class="content"><b style="font-size: 15px;">{breach_name}</b><br>{description}
+        </p>\n""".format(
+            breach_name=row[0], description=row[1]
         )
+    html += "\n</div>"
+    return html
+
+
+def credential(chevron_dict, start_date, end_date, org_uid):
+    """Build exposed credential page."""
+    Credential = Credentials(start_date, end_date, org_uid)
+    total = Credential.total()
+    breach = Credential.breaches()
+    pw_creds = Credential.password()
+    ce_date_df = Credential.by_days()
+    breach_det_df = Credential.breach_details()
+    breach_appendix = Credential.breach_appendix()
+    # Build exposed credential stacked bar chart
+    width = 24
+    height = 9.5
+    name = "inc_date_df"
+    title = "Reported Exposures by Day"
+    x_label = "Date Reported"
+    y_label = "Creds Exposed"
+    cred_date_chart = Charts(
+        ce_date_df,
+        width,
+        height,
+        name,
+        title,
+        x_label,
+        y_label,
+    )
+    cred_date_chart.stacked_bar()
+    breach_table = buildTable(breach_det_df, ["table"])
+
+    creds_dict = {
+        "breach": breach,
+        "creds": total,
+        "pw_creds": pw_creds,
+        "breach_table": breach_table,
+        "breachAppendix": buildAppendixList(breach_appendix),
+    }
+    chevron_dict.update(creds_dict)
+
+    return chevron_dict, Credential.query_hibp_view, Credential.query_cyberSix_creds
+
+
+def masquerading(chevron_dict, start_date, end_date, org_uid):
+    """Build masquerading page."""
+    Domain_Masq = Domains_Masqs(start_date, end_date, org_uid)
+    summary = Domain_Masq.summary()
+    domain_count = Domain_Masq.count()
+    utlds = Domain_Masq.utlds()
+
+    domain_table = buildTable(summary, ["table"], [])
+    chevron_dict.update(
+        {
+            "domain_table": domain_table,
+            "suspectedDomains": domain_count,
+            "uniqueTlds": utlds,
+        }
```

Comment:
Similar to my previous comment, I'd remove these single-use variables:
```suggestion
    chevron_dict.update(
        {
            "domain_table": buildTable(Domain_Masq.summary(), ["table"], []),
            "suspectedDomains": Domain_Masq.count(),
            "uniqueTlds": Domain_Masq.utlds(),
        }
```

Reply:
Addressed [here](https://github.com/cisagov/pe-reports/pull/114/commits/a6226dc399178094a6c6f9fcd5d068d611859d5e).

Reply:
`domain_table` is still a single-use variable and can be eliminated via:
```
"domain_table": buildTable(Domain_Masq.summary(), ["table"], [])
```

Reply:
@dav3r addressed in the following commit: [3228942](https://github.com/cisagov/pe-reports/pull/114/commits/32289424226c4801893f4e99c040e807a406713a)